.github/workflows/ci-base.yml

```diff
@@ -0,0 +1,44 @@
+name: PR Releases
+
+on:
+  workflow_call:
+    inputs:
+      os:
+        description: 'Operating System to run the workflow on'
+        required: true
+        type: string
+      node-version:
+        description: 'Node.js version to use'
+        required: true
+        type: string
+      publish-extra-args:
+        description: 'Additional arguments for publishing packages'
+        required: false
+        type: string
+        default: ''
+
+jobs:
+  ci-releases:
+    name: Package Releases
+    # Confirm if l10nbot check is still relevant.
+    if: ${{ github.actor != 'l10nbot' && github.repository_owner == 'mui' }}
```

Comment:
Any comments here ?

Reply:
yeah, in the future if a specific project needs more filtering, I believe ir's also possible to add a conditional when calling this workflow, right? Perhaps repository_owner we can keep?

Reply:
Yeah. The checks can be added by the user of this workflow as well. I kept owner check so that checks in the repos can be avoided.